SUPPORTED_OPS.rst

```diff
@@ -0,0 +1,70 @@
+Supported Operations
+####################
+
+.. table::
+
+    +------------------+-----------------------------------------+--------------+-----------------------+
+    | **Visibility**   | **Operation**                           | **Protocol** | **Number of Parties** |
+    +==================+=========================================+==============+=======================+
+    | Public & Private | Addition                                | - Falcon     | - 3                   |
```

Comment:
Q: I am not sure here if we should specify all the protocols for the operations that are implemented the same way -- like the additions/subtraction is the same for all those protocols

Reply:
True, since they rely on sharetensor and Rstensor.

Reply:
Maybe we could mention that we support sharetensor that allows N parties that hold 1 share per party and RStensor which holds N-1 shares per party.

Reply:
Okey. We could create two tables. One table to show which protocols use which kind of tensor, we could add the information about the shares above. Then, we only need to create another table to show which operations support each kind of tensor.

What do you think @kamathhrishi.

Reply:
@kamathhrishi, ping

Reply:
@danielorihuela That sounds like a great idea. Go ahead with it :)

Reply:
Perfect, @gmuraru @kamathhrishi I isolated the addition and substraction operations into another table. You can check it [here](https://github.com/danielorihuela/SyMPC/blob/docs/add-supported-operations-protocols-and-security-guarantees-to-readme/SUPPORTED_OPS.rst).